src/main/java/com/three_tech_solutions/slot_app/controllers/implementations/StudentControllerImpl.java

```diff
@@ -44,4 +50,9 @@ public void activateStudent(UUID studentId) {
     public StudentResponse updateStudent(UUID studentId, UpdateStudentRequest studentUpdated){
         return studentService.updateStudent(studentId, studentUpdated);
     }
+
+    @Override
+    public List<StudentMonthlyFeeResponseDto> getStudentMonthlyFees(UUID studentId, String month, LocalDate expirationDate, MonthlyFeeStatus status) {
+        return monthlyFeeService.getMonthlyFeesByStudent(studentId, month, expirationDate, status);
```

Comment:
Es por eso que acá, en vez de llamar al monthlyFeeService, deberiamos llamar al studentService


```suggestion
        return studentService.getStudentMonthlyFees(
                studentId,
                month,
                expirationDate,
                status
        );
```